drivers/telescope/lx200driver.cpp

```diff
@@ -562,19 +571,35 @@ int getSiteLatitude(int fd, int *dd, int *mm)
 
     DEBUGFDEVICE(lx200Name, DBG_SCOPE, "RES <%s>", read_buffer);
 
-    if (sscanf(read_buffer, "%d%*c%d", dd, mm) < 2)
+    *ssf = 0.0;
+    if (sscanf(read_buffer, "%d%*c%d:%lf", dd, mm, ssf) < 2)
```

Comment:
Shouldn't this be %.1lf? Only one decimal place?

Reply:
well because the third parameter is completely optional I did not want to limit significance there. If there's a ss.ss then it can be used for rounding at least of the last .s digit. So I deliberately left out the 1. Does this make sense to you ?

Reply:
Right! Yes makes sense